src/invidious/comments.cr

```diff
@@ -587,18 +587,18 @@ def content_to_comment_html(content)
           end
         end
 
-        text = %(<a href="#{url}">#{text}</a>)
+        text = %(<a href="#{url}">#{reduce_uri(url)}</a>)
       elsif watch_endpoint = run["navigationEndpoint"]["watchEndpoint"]?
         length_seconds = watch_endpoint["startTimeSeconds"]?
         video_id = watch_endpoint["videoId"].as_s
 
         if length_seconds && length_seconds.as_i > 0
           text = %(<a href="javascript:void(0)" data-onclick="jump_to_time" data-jump-time="#{length_seconds}">#{text}</a>)
```

Comment:
You forgot that one ^^

Reply:
For the link to a section of the video you think it's interesting to reduce it too?

Reply:
* `www.youtube.com` = 15 chars
* `/watch?v=` = 9
* a video id is 11 chars
* `&t=...` = from 4 to 12 chars (`10h25m32s` = 9 chars)

So the total is 47 characters. So it won't be needed :)